flagsmith/flagsmith.py

```diff
@@ -322,6 +330,7 @@ def _get_environment_flags_from_document(self) -> Flags:
         if self._environment is None:
             raise TypeError("No environment present")
         return Flags.from_feature_state_models(
+            # TODO: Use from_evaluation_result
```

Comment:
@khvn26.

Can I safely assume that `FlagResult` can be manipulated the same way as a feature_state?